lib/install/action_cable.rb

```diff
@@ -0,0 +1,144 @@
+# frozen_string_literal: true
+
+require "cable_ready/installer"
+
+# verify that Action Cable is installed
+if defined?(ActionCable::Engine)
+  say "✅ ActionCable::Engine is required and in scope"
+else
+  halt "ActionCable::Engine is not required. Please add or uncomment `require \"action_cable/engine\"` to your `config/application.rb`"
```

Comment:
This ("is not required") could be misunderstood. Suggestion:

```suggestion
  halt "ActionCable::Engine is not loaded. Please add or uncomment `require \"action_cable/engine\"` to your `config/application.rb`"
```